src/tasks/processBulkUpload.ts

```diff
@@ -178,7 +178,6 @@ const createApplicationFormFieldsForBulkUpload = async (
 				applicationFormId,
 				baseFieldId: baseField.id,
 				position: index,
-				label: baseField.label,
```

Comment:
As @slifty said, we do need to keep application form field labels. However, `baseField.label` no longer exists.

I think what we want to do is look for an English label - because, in practice, at least for the foreseeable future, they'll all have English labels - and use that here; if there is no English label, we can use the short code as the application form field label and log a warning.